packages/astro/e2e/view-transitions.test.js

```diff
@@ -782,7 +786,7 @@ test.describe('View Transitions', () => {
 	});
 
 	test('client:only styles are retained on transition (1/2)', async ({ page, astro }) => {
-		const totalExpectedStyles = 9;
+		const totalExpectedStyles = 8;
```

Comment:
Need to confirm this is correct, just want to see if the rest of CI passes with this change first.